NeuroID/src/debug/java/com/neuroid/tracker/storage/NIDDataStoreManager.kt

```diff
@@ -260,4 +173,91 @@ private object NIDDataStoreManagerImp : NIDDataStoreManager {
 
         return jsonBody.toString()
     }
+
+    /**
+     * Convert a list of NIDEventModel to a list of JSONObject.  The provided list will be cleared
+     * during the conversion.  Where necessary, events are updated to fill in missing values.
+     */
+    private fun toJsonList(list: MutableList<NIDEventModel>): List<JSONObject> {
+        val jsonList = mutableListOf<JSONObject>()
+        while (list.isNotEmpty()) {
+            val event = list.removeAt(0)
+            event.accel?.let {
+                it.x = it.x ?: NIDSensorHelper.valuesAccel.axisX
```

Comment:
these are returning the same values every time. ill push an update shortly to fix this.

Reply:
fixed